src/apps/activity_flows/domain/create.py

```diff
@@ -0,0 +1,16 @@
+import pydantic.types as types
+from pydantic import BaseModel
+
+__all__ = ["ActivityFlowCreate", "ActivityFlowItemCreate"]
+
+
+class ActivityFlowItemCreate(BaseModel):
+    activity_guid: types.UUID4
+
+
+class ActivityFlowCreate(BaseModel):
+    name: str
+    description: types.Dict[str, str]
+    is_single_report: bool = False
+    hide_badge: bool = False
+    items: types.List[ActivityFlowItemCreate]
```

Comment:
The same about the typing as above. Fell free to resolve all of them after replying at least in one

Reply:
?

Reply:
Ohh don't see my message for some reason...
We use pydantic and it works perfect with python built-in types system.

```suggestion

class ActivityFlowCreate(BaseModel):
    name: str
    description: dict[str, str]
    is_single_report: bool = False
    hide_badge: bool = False
    items: list[ActivityFlowItemCreate]
```

Reply:
I got it